### PR TITLE
[HS-1526] Add tests for Summary page functionality, add small fixes to other scenarios

### PR DESCRIPTION
**Pull request by bahka**
- Fix naming problems in existed scenarios
- Add more specific locators for error texts
- Add PO, Steps and Scenarios for Summary page

## Jira link(s)
- https://issues.opennms.org/browse/HS-1526

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

File: external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/steps/cloud/CloudLoginSteps.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package org.opennms.horizon.systemtests.steps.cloud;
```
Comment (chuklov) [inline external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/steps/cloud/CloudLoginSteps.java:1]: Missing header here.

File: external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/steps/portal/EditInstanceNameSteps.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package org.opennms.horizon.systemtests.steps.portal;
```
Comment (chuklov) [inline external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/steps/portal/EditInstanceNameSteps.java:1]: Missing header

File: external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/steps/portal/PrepareDataSteps.java
Comment (chuklov) [inline external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/steps/portal/PrepareDataSteps.java:1]: Missing header

File: external-it/hs-system-tests/src/test/resources/portal-features/delete-instance.feature [hunk 3]
```diff
@@ -12,7 +12,7 @@ Feature: Administrator can delete an instance
     And the IT Administrator sees the 'Cloud Instance Details' page for the "CLOUD-T521" instance
 
   @TestCaseKey=CLOUD-T522
-  Scenario: IT Administrator cancels deleting a BTO instance using button
+  Scenario: IT Administrator cancels deleting a BTO instance using X
```
Comment (chuklov) [inline external-it/hs-system-tests/src/test/resources/portal-features/delete-instance.feature:15]: To be consistent maybe we should have here X button as in edit-instance.feature file?

File: external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/pages/portal/EditInstanceNamePopup.java [hunk 4]
```diff
@@ -1,8 +1,10 @@
 package org.opennms.horizon.systemtests.pages.portal;
```
[the remark this answers is not in the document]
Comment (bahka) [reply external-it/hs-system-tests/src/test/java/org/opennms/horizon/systemtests/pages/portal/EditInstanceNamePopup.java:1]: I already added it in another PR, is it oK? 
> https://github.com/OpenNMS-Cloud/horizon-stream/pull/1061/files

File: external-it/hs-system-tests/src/test/resources/portal-features/delete-instance.feature (hunk 3 above)
Comment (bahka) [reply external-it/hs-system-tests/src/test/resources/portal-features/delete-instance.feature]: added everywhere

**Review by chuklov — APPROVED** [review]
LGMT

**Pull request merged**